src/components/Shared/RPBGridToolbar.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+import GridToolbar from 'src/components/Shared/GridToolbar'
+import { useWindow } from 'src/windows'
+import ReportParameterBrowser from 'src/components/Shared/ReportParameterBrowser'
+import { Grid } from '@mui/material'
+
+const RPBGridToolbar = ({ add, access, onApply, reportName, onSearchClear, doSearch = true, ...rest }) => {
+  const { stack } = useWindow()
+  const [paramsArray, setParamsArray] = useState([])
+  const [search, setSearch] = useState([])
+
+  const openRPB = () => {
+    stack({
+      Component: ReportParameterBrowser,
+      props: {
+        reportName: reportName,
+        paramsArray,
+        setParamsArray
+      },
+      width: 700,
+      height: 500,
+      title: 'Report Parameters Browser'
+    })
+  }
+
+  const actions = [
+    {
+      key: 'OpenRPB',
+      condition: true,
+      onClick: openRPB,
+      disabled: false
+    },
+    {
+      key: 'GO',
+      condition: true,
+      onClick: () => onApply({ search, paramsArray }),
+      disabled: false
+    }
+  ]
+
+  return (
+    <GridToolbar
+      onAdd={add}
```

Comment:
this shouldn't be passed directly. We're already passing rest to the component. 
onAdd
maxAccess